### ISSUE TITLE
🐛 Bug Report: Redirection to console on Password recover/reset

### ISSUE THREAD

**Issue by Abhishek-90** [appwrite/appwrite#5758]
### 👟 Reproduction steps

When I click on the password recover/reset link sent to my email, once the recover page is loaded, within one second, I am redirected to the console page. This bug is not allowing me to change my password.

### 👍 Expected behavior

I shouldn't be redirected to appwrite console and allowed to successfully change my password.

### 👎 Actual Behavior

Getting redirected to console page and not able to change my password.

### 🎲 Appwrite version

Appwrite Cloud

### 💻 Operating system

Windows

### 👀 Have you spent some time to check if this issue has been raised before?

- [X] I checked and didn't find similar issue

### 🏢 Have you read the Code of Conduct?

- [X] I have read the [Code of Conduct](https://github.com/appwrite/.github/blob/main/CODE_OF_CONDUCT.md)

**Comment by joeyouss** [comment]
Hi @Abhishek-90 , can you let me know if you have logged out first?

**Comment by Abhishek-90** [comment]
> Hi @Abhishek-90 , can you let me know if you have logged out first?

No @joeyouss, I didn't logout first. I had used github to sign up for appwrite. I didn't know my appwrite password or tried to change it but when I clicked on the link sent by appwrite on my email, it redirected to appwrite console before I could change my password.

**Comment by NiazMorshed2007** [comment]
Same happened to me

**Comment by TheJazzDev** [comment]
@Abhishek-90 Not saying that process is cool but since you didn't sign up using email and password, why would you want to reset password? You will always use your github to sign up and as long as you have access to that github account, you don't have a problem.

**Comment by Abhishek-90** [comment]
@TheJazzDev I agree, logging into my appwrite account won't be an issue since I signed up using github, but certain operations like deleting an appwrite project requires me to add password for confirmation, at that point, it becomes an issue. I tried using my github password but it didn't work.

**Comment by joeyouss** [comment]
So, for login via Github: https://github.com/appwrite/appwrite/issues/5509

If you've set a password, hopefully it won't occur anymore. We have made a PR to fix the issue mentioned here though!

**Comment by ItzNotABug** [comment]
This is probably still not fixed.

**Comment by HarshMN2345** [comment]
Closing the premature redirect report as fixed in the new Console
